### ISSUE TITLE
prettier-plugin-solidity fails on ethereum-optimism/contracts

### ISSUE THREAD

**Issue by janek26** [prettier-solidity/prettier-plugin-solidity#451]
If you try to use the Plugin to format [this file](https://github.com/ethereum-optimism/contracts/blob/77d30d66b93fa65782d2f15d07843822b28df9e6/contracts/optimistic-ethereum/iOVM/chain/iOVM_CanonicalTransactionChain.sol) you get the following error thrown:

```
contracts/optimistic-ethereum/iOVM/chain/iOVM_CanonicalTransactionChain.sol
[error] contracts/optimistic-ethereum/iOVM/chain/iOVM_CanonicalTransactionChain.sol: Error: Comment "uint40 _shouldStartAtElement," was not printed. Please report this error!
[error]     at /Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/index.js:14350:13
[error]     at Array.forEach (<anonymous>)
[error]     at Object.ensureAllCommentsPrinted (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/index.js:14348:15)
[error]     at coreFormat (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/index.js:14912:12)
[error]     at format (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/index.js:15131:14)
[error]     at Object.formatWithCursor (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/index.js:57542:12)
[error]     at format (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/bin-prettier.js:15883:14)
[error]     at Object.formatFiles (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/bin-prettier.js:16067:16)
[error]     at Object.run (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/bin-prettier.js:16538:14)
[error]     at Object.<anonymous> (/Users/janek.rahrt/Projects/optimism/contracts/node_modules/prettier/bin-prettier.js:16553:5)
```

It seems to be due to [Line 213](https://github.com/ethereum-optimism/contracts/blob/77d30d66b93fa65782d2f15d07843822b28df9e6/contracts/optimistic-ethereum/iOVM/chain/iOVM_CanonicalTransactionChain.sol#L213)

**Comment by fvictorio** [comment]
Thanks for reporting this @janek26! Definitely a bug on our side.

For the time being, you can use this (admittedly ugly) workaround:

```solidity
    function /* prettier-ignore */ appendSequencerBatch(
```

The rest of the contracts seem to work fine (although of course, if you spot something formatted in an obviously wrong way, let us know!)

**Comment by fvictorio** [comment]
Closed by #452.
